lib/tjbot.js

```diff
@@ -774,28 +786,78 @@ TJBot.prototype.read = function() {
     var self = this;
 
     return new Promise(function(resolve, reject) {
-        temp.open('tjbot', function(err, info) {
+        self.takePhoto().then(function(filePath) {
+            resolve(self.describePhotoText(filePath))
+        });
+    });
+}
+
+/*
+Describe text in photo by sending it to the Watson Visual Recognition Services.
+ */
+TJBot.prototype.describePhotoText = function(filePath) {
```

Comment:
i'm not totally happy with the name `describePhotoText`.  It's not really about _describing_, it's more about _reading_ or _recognizing_.  The Watson API calls it `recognizeText`, so should this method be called `recognizeTextInPhoto()` ?  (and then, instead of `describePhoto()` above we should call it `recognizeObjectsInPhoto()`?)